TrkFilters/src/HelixFilter_module.cc

```diff
@@ -95,6 +101,13 @@ namespace mu2e
     return true;
   }
 
```

Comment:
You could make the parameter const
int HelixFilter::evalIPAPresc(const float &phi0){